Update teleporting.md (#402)
Roblox/creator-docs: content/en-us/projects/teleporting.md

### Teleporting to Specific Servers

> To teleport users to specific servers, set the target server using `Class.TeleportOptions` and pass it to the `Class.TeleportService:TeleportAsync()` method. If you don't specify a server, users are sorted into any public server. The details of the first user in the list is used to matchmake to a public server.

Reviewer: ```suggestion
To teleport users to specific servers, set the target server using `Class.TeleportOptions` and pass it to the `Class.TeleportService:TeleportAsync()` method. If you don't specify a server, users are teleported into a matchmade public server. The information of the first user in the list is used to matchmake to that public server.
```
